src/komponenter/alleredeOpprettetTiltak/api/AlleredeOpprettetAvtaleProvider.tsx

```diff
@@ -24,6 +28,21 @@ export const AlleredeOpprettetAvtaleContext = React.createContext<Context>({
     alleredeRegistrertAvtale: initAlleredeOpprettetInfo,
 } as Context);
 
+export const useAlleredeOpprettetAvtale = () => useContext(AlleredeOpprettetAvtaleContext);
+
+const endrePathAlleredeOpprettet = (
+    pathnameList: string[],
+    path: Path,
+    alleredeOpprettet: AlleredeOpprettetInfo,
+    setAlleredeOpprettet: Dispatch<SetStateAction<AlleredeOpprettetInfo>>,
+) => {
+    if (pathnameList.includes(path)) {
+        if (alleredeOpprettet.steg !== path) {
+            setAlleredeOpprettet({ avtaler: [], deltaker: '', steg: path });
+        }
+    }
+};
```

Comment:
Dette er flyttet kode, men må uansett påpeke at jeg ikke helt skjønner hva hensikten med dette er!

Sjekker vi om en avtale er opprettet ved å lagre en liste over avtaler som er opprettet i en context-dings i frontenden..?

Reply:
Åja, dette er vel eksisterende avtaler for en deltaker da kanskje..?

Reply:
Helt ærlig, vet ikke hva den gjør. 🤷 Men funksjonen ble bare brukt her, så derfor flyttet jeg den fra `alleredeUtils` slik at `alleredeUtils` kunne slettes.